src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/permute_concat_concat_permute.hpp

```diff
@@ -23,6 +23,16 @@ class PermuteConcatConcatPermute : public testing::WithParamInterface<PermuteCon
 public:
     static std::string getTestCaseName(const testing::TestParamInfo<PermuteConcatConcatPermuteTuple>& obj);
 
+    void ExpectLoadNetworkToThrow() {
```

Comment:
Cannot we reuse this function in more tests?

Reply:
I can create a separate class to be inherited by specific tests as adding this method to LayerTestsCommon is rather not a good idea.

Reply:
I think it's good to avoid code duplication.

Reply:
Finally decided to add it in LayerTestsCommon.